cmake/ExternalDependencies.cmake

```
@@ -29,7 +29,7 @@ endif()
 # cmake-format: off
 set(MQT_CORE_VERSION 2.5.1
```

```suggestion
set(MQT_CORE_VERSION 2.5.2
```
Given that this is no longer pointing to the commit that is tagged as `2.5.1`, this should be bumped as well. Otherwise, the next update run will compare the versions and overwrite the rev with the `2.5.1` release one.